external-crates/move/crates/move-command-line-common/src/files.rs

```diff
@@ -41,7 +41,7 @@ pub const MOVE_IR_EXTENSION: &str = "mvir";
 /// Extension for Move bytecode files
 pub const MOVE_COMPILED_EXTENSION: &str = "mv";
 /// Extension for Move source map files (mappings from source to bytecode)
-pub const SOURCE_MAP_EXTENSION: &str = "mvsm";
+pub const DEBUG_INFO_EXTENSION: &str = "mvdbg";
```

Comment:
We call good with `mvdbg`?

Reply:
Clearly, `mvdbg` is just a proposal, but seems at least somewhat consistent with the other extensions. Very open to other proposals, though!